### PR TITLE
Add sphinx class docs to configuration.

### LINKED ISSUE #177: Update comments to provide sphinx markup
E.g. so the object documentation shows on Read the Docs.

### PR DESCRIPTION
**Pull request by pmtischler**
Fixes #177

### CODE REVIEW

File: server/auvsi_suas/models/mission_config.py [hunk 1]
```diff
@@ -27,7 +27,25 @@
...
+        targets: The judge created targets.
+
+        emergent_last_known_pos: The last known position of the emergent target.
+        off_axis_target_pos: Off-axis target position.
+        sric_pos: The SRIC position.
+        air_drop_pos: The air drop position.
+        stationary_obstacles: The stationary obstacles.
+        moving_obstacles: The moving obstacles.
+    """
     # Whether the mission is active. Only one mission can be active at a time.
```
Comment (prattmic) [inline server/auvsi_suas/models/mission_config.py]: Did you mean to remove these?

File: server/auvsi_suas/models/mission_config.py [hunk 2]
```diff
@@ -27,7 +27,25 @@
...
+    Attributes:
+        is_active: Whether the mission is active. Only one mission can be
+            active at a time.
+        home_pos: The home position for use as a reference point. Should be the
+            tents.
+        fly_zones: Valid areas for the UAS to fly.
+        mission_waypoints: The waypoints that define the mission waypoint path
+        search_grid_points: The polygon that defines the search grid.
+        targets: The judge created targets.
+
```
Comment (prattmic) [inline server/auvsi_suas/models/mission_config.py]: Why the extra newline?

File: server/auvsi_suas/models/target.py [hunk 3]
```diff
@@ -125,55 +125,45 @@ class Color(Choices):
...
+        alphanumeric: Target alphanumeric.
+        alphanumeric_color: Target alphanumeric color.
+        description: Free-form target description.
+        autonomous: Target is an ADLC submission.
+        thumbnail: Uploaded target image thumbnail.
+        thumbnail_approved: Whether judge considers thumbnail valid for target.
+        creation_time: Time that this target was first created.
+            auto_now_add automatically sets this field on initial creation.
+        last_modified_time: Time that this target was last modified.
+            auto_now automatically updates this field on save().
```
Comment (prattmic) [inline server/auvsi_suas/models/target.py]: These notes may make more sense left with the definition.

File: server/auvsi_suas/models/mission_config.py (hunk 2 above)
Comment (pmtischler) [inline server/auvsi_suas/models/mission_config.py]: Typo. Fixed

File: server/auvsi_suas/models/mission_config.py (hunk 1 above)
Comment (pmtischler) [inline server/auvsi_suas/models/mission_config.py]: Yup, fixed.

File: server/auvsi_suas/models/target.py (hunk 3 above)
Comment (pmtischler) [inline server/auvsi_suas/models/target.py]: Removed. I actually don't think we need to include that at all, as it's documenting Django.

**Pull request merged**